src/platform_impl/windows/event_loop.rs

```diff
@@ -1443,36 +1454,36 @@ unsafe fn public_window_callback_inner<T: 'static>(
             0
         }
 
-        winuser::WM_CAPTURECHANGED => {
+        WM_CAPTURECHANGED => {
             // lparam here is a handle to the window which is gaining mouse capture.
             // If it is the same as our window, then we're essentially retaining the capture. This
             // can happen if `SetCapture` is called on our window when it already has the mouse
             // capture.
-            if lparam != window as isize {
+            if lparam != window {
                 userdata.window_state.lock().mouse.capture_count = 0;
             }
             0
         }
 
-        winuser::WM_TOUCH => {
-            let pcount = LOWORD(wparam as DWORD) as usize;
+        WM_TOUCH => {
+            let pcount = wparam & 0xffff; // LOWORD
             let mut inputs = Vec::with_capacity(pcount);
-            inputs.set_len(pcount);
-            let htouch = lparam as winuser::HTOUCHINPUT;
-            if winuser::GetTouchInputInfo(
+            let htouch = lparam;
+            if GetTouchInputInfo(
                 htouch,
-                pcount as UINT,
+                pcount as u32,
                 inputs.as_mut_ptr(),
-                mem::size_of::<winuser::TOUCHINPUT>() as INT,
-            ) > 0
+                mem::size_of::<TOUCHINPUT>() as i32,
+            ) == true.into()
             {
+                inputs.set_len(pcount);
                 for input in &inputs {
                     let mut location = POINT {
                         x: input.x / 100,
                         y: input.y / 100,
                     };
 
-                    if winuser::ScreenToClient(window, &mut location as *mut _) == 0 {
+                    if !ScreenToClient(window, &mut location) == true.into() {
```

Comment:
Semantic change (negative comparison to `true` instead of positive comparison to `false`)